lib/ansiblelint/__init__.py

```diff
@@ -140,8 +140,10 @@ def matchyaml(self, file, text):
 
 class RulesCollection(object):
 
-    def __init__(self):
+    def __init__(self, rulesdirs=[]):
```

Comment:
Please don't use mutable defaults. In Python, it's common to initialize such things in the init body:
```suggestion
    def __init__(self, rulesdirs=None):
        if rulesdirs is None:
            rulesdirs = []
```